config/dev.exs

```diff
@@ -7,4 +7,8 @@ config :argos,
 
   chronontology_url: "https://chronontology.dainst.org/data",
   chronontology_batch_size: 10,
-  chronontology_harvest_interval: 1000 * 10 # 10 seconds
+  chronontology_harvest_interval: 1000 * 10, # 10 seconds
+
+  gazetteer_url: "https://gazetteer.dainst.org/",
```

Comment:
Ich glaube vom Gezetteer läuft eine Testversion unter https://gazetteer.dainst.org/gazetteer-test

Ich würde die eventuell vorziehen zumindest fürs dev-Environment. (Bei chronontology scheint es keine Testversion zug geben.)

Und, nicht super wichtig, aber ich würde den Trailing Slash hinter der URL weglassen oder alternativ bei allen anderen URLs einen anfügen, so dass wir das einheitlich haben.

Reply:
Wegen der Testversion: Ich würde davon eher abraten: Mit SD-860 sollen am Ende die Daten integriert werden. Das heißt wenn ich nach der arabischen Bezeichnung eines Ortes suche, möchte ich (auch) die Projekte die mit dem Ort verknüpft sind. Wenn wir jetzt auf die Testinstanz des Gazetteers gehen, müsste deine Projekte-Instanz auch auf die Testinstanz des Gazetteers gehen...

Reply:
Der Test-Gazetteer benutzt die selbe Datenbank wie der normale Gazetteer. Der wurde früher nur dazu verwendet, um neue Features mit den Originaldaten zu testen und anderen zu zeigen, bevor sie in die Produktivversion gehen. Es sollte am Ende also eigentlich keinen Unterschied machen, was ihr da nehmt. Abgesehen von der URL (die auch in den JSON-Daten als Teil der Orts-IDs zurückgegeben wird) sind die Versionen identisch. Ich würde vorschlagen, einfach die normale Gazetteer-URL zu nehmen, weil die Testversion sowieso nicht mehr wirklich benutzt wird.

Reply:
Ok, einverstanden. Den Trailing Slash würde ich trotzdem noch wegmachen ;)

Reply:
Slash ist entfernt.